website/content/v1.12/talos-guides/install/local-platforms/qemu.md

```diff
@@ -0,0 +1,328 @@
+---
+title: QEMU
```

Comment:
the documentation got moved out to `siderolabs/docs`, so probably we can/should do another PR there for 1.12